frontend/src/components/common/SortMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import {
+  Flex,
+  Button,
+  Radio,
+  VStack,
+  Text,
+  RadioGroup,
+  Popover,
+  PopoverTrigger,
+  PopoverBody,
+  PopoverContent,
+  PopoverFooter,
+  Divider,
+  HStack,
+} from "@chakra-ui/react";
+import { FilterOptionsIcon } from "../../assets/icons";
+
+interface SortMenuProps {
+  tableProperties: string[];
+  onSortProperty: React.Dispatch<React.SetStateAction<string>>;
+  onSortOrder: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const SortMenu = ({
+  tableProperties,
+  onSortProperty,
+  onSortOrder,
+}: SortMenuProps): React.ReactElement => {
+  const [sortProperty, setSortProperty] = React.useState("firstName");
+  const [sortOrder, setSortOrder] = React.useState("ascending");
+
+  const propertyList = (
+    <RadioGroup
+      color="blue.300"
+      onChange={(e) => setSortProperty(e)}
+      value={sortProperty}
+    >
+      <VStack alignItems="left" gap={1}>
+        <Text pb="2" textStyle="link">
+          Property
+        </Text>
+        {tableProperties.map((property, index) => (
+          <Radio defaultChecked={index === 1} key={index} value={property}>
```

Comment:
should this be 0? but also, i removed this attribute and it was same behaviour - so i'm not sure if it's actually doing anything ?? lol

Reply:
good catch! not needed